android/sdk/src/main/java/org/jitsi/meet/sdk/BaseReactView.java

```diff
@@ -109,16 +110,17 @@ public static BaseReactView findViewByExternalAPIScope(
 
     public BaseReactView(@NonNull Context context) {
         super(context);
+        initialize((Activity)context);
```

Comment:
The behaviour here is kept, I've just separated in into a method, so it can be called from all constructors. Indeed there might be situations where this would crash, but this was there before.

Reply:
Ah, good to know!